manager/controlapi/service_test.go

```diff
@@ -38,6 +38,40 @@ func createSpec(name, image string, instances uint64) *api.ServiceSpec {
 	}
 }
 
+func createSpecWithDuplicateMounts(name string) *api.ServiceSpec {
+	return &api.ServiceSpec{
+		Annotations: api.Annotations{
```

Comment:
Rather than copy pasting, just use the service from the other method and add mounts.